documentation/site/content/product/roadmap/next.md

```diff
@@ -1,7 +1,7 @@
 ## Next ➡️
 
-* improving how early career teachers are transferred between different schools, so that all users have the information they need and can make changes easily
 * considering how to indicate when early career teachers have left for other reasons, so we can reduce confusion for schools and lead providers
-* start building support tooling to help with registration, so that our support users have the autonomy they need to quickly and simply aid schools
-* building a service for appropriate bodies to submit induction data, so that we can maintain induction processes and consolidate our data 
-* getting ready for the changes in policy for 2025, so that our digital services are up to date and can maintain registration of ECTs
+* start work on the API for lead providers, so we can send them information from schools and pay them for providing training
+* improve how the API works, so lead providers can submit the data they need and have it validated correctly, saving them time and improving our data quality
+* building a check using this API to make sure the migrated data is correct
+* building a way for appropriate bodies to efficiently record inductions in bulk and reduce their workload
```

Comment:
Really minor but the leading words are inconsistent:

>  [we are going to] _start work on_ the API... 

> [we will be] _building a check_ using...

Would a lead-in sentence below the subheading help?